perl6.html.markdown

```diff
@@ -798,19 +798,39 @@ class Item does PrintableVal {
 
 ### Exceptions
 # Exceptions are built on top of classes, in the package `X` (like `X::IO`).
-# You can access the last exception with the special variable `$!`
-# (use `$_` in a `CATCH` block) Note: This has no relation to $!variables.
+# In Perl6 exceptions are automatically 'thrown'
+open 'foo' #> Failed to open file foo: no such file or directory
+# It will also print out what line the error was thrown at and other error info
 
-# You can throw an exception using `die`:
-open 'foo' or die 'Error!'; #=> Error!
+# You can throw your own exceptions using `die`:
+die 'Error!'; #=> Error!
 # Or more explicitly:
 die X::AdHoc.new(payload => 'Error!');
 
+# In Perl 6, `orelse` is similar to the `or` operator, except it only matches
+# `undef` instead of anything evaluating as false.
```

Comment:
Perl 6 doesn't have `undef`. "it matches exception instead of falsy values" should be correct